packages/opentelemetry-core/test/common/trace/tracestate.test.ts

```diff
@@ -105,7 +115,7 @@ describe('TraceState', () => {
           .map((_: null, num: number) => `a${num}=${num}`)
           .join(',')
       );
-      assert.deepStrictEqual(state['_keys']().length, 32);
+      // assert.deepStrictEqual(state['_keys']().length, 32);
```

Comment:
```suggestion
      assert.deepStrictEqual(state.serialize().split(',').length, 32);
```
The (uncommented) test as-is gives TypeScript errors. It can now be replaced with the above to assert length of the array.